client/src/templates/Challenges/classic/lower-jaw.tsx

```diff
@@ -166,7 +177,7 @@ const LowerJaw = ({
       return null;
     }
   };
-
+  const isBlockCompleted = 100;
```

Comment:
This should be a global variable, so it doesn't render everytime the component is mounted

Reply:
ok updating